bin/logstash.bat

```diff
@@ -52,7 +52,7 @@ for %%i in ("%LS_HOME%\logstash-core\lib\jars\*.jar") do (
 	call :concat "%%i"
 )
 
-%JAVA% %JAVA_OPTS% -cp "%CLASSPATH%" org.logstash.Logstash %*
+%JAVA% %JAVA_OPTS% -cp %CLASSPATH% org.logstash.Logstash %*
```

Comment:
this will revert back and we potentially re-introduce issues fixed at https://github.com/elastic/logstash/pull/9898